main.go

```diff
@@ -50,6 +52,15 @@ func init() {
 func decode(conn *twitterstream.Connection) {
 	for {
 		if tweet, err := conn.Next(); err == nil {
+
+			// Filter out ignored keywords
+			for _, keyword := range strings.Split(config.KeywordsIgnored, ",") {
+				if strings.Contains(tweet.Text, keyword) {
+					log.Printf("Tweet contained ignored keyword: %s", keyword)
+					return
```

Comment:
This decodes an individual tweet, so we want to stop if that tweet has an ignored keyword, right?